src/flatpickr.directive.ts

```diff
@@ -437,6 +437,10 @@ export class FlatpickrDirective
     });
     options.time_24hr = options.time24hr;
 
+    if (typeof this.convertModelValue === 'undefined') {
+      this.convertModelValue = this.defaults.convertModelValue;
```

Comment:
It's generally recommended not to mutate angular inputs like this, can we add it in a getter like:

```
get convertModelValueEnabled() {
  return typeof this.convertModelValue === 'undefined' ? this.defaults.convertModelValue : this.convertModelValue;
}
```

and then change  `this.convertModelValue` -> `this.convertModelValueEnabled` in the 2 places we reference it